src/ui/zif_abapgit_popups.intf.abap

```diff
@@ -100,6 +100,7 @@ INTERFACE zif_abapgit_popups
   METHODS popup_transport_request
     IMPORTING
       !is_transport_type  TYPE zif_abapgit_definitions=>ty_transport_type
+      !iv_use_default     TYPE abap_bool DEFAULT abap_false
```

Comment:
is this default correct? looks like the default behavior is changed?

Reply:
#5453